### PR TITLE
[11.0][FIX] purchase_open_qty: not able to filter by pending/receiving quan…

### PR DESCRIPTION
**Pull request by eantones**
…tity at line level

Wrong filter fields at line level, the module is trying to use the
computed fields of the order instead of the line ones.

### CODE REVIEW

**Comment by eantones** [comment]
Hi @JordiBForgeFlow @MiquelRForgeFlow @AdriaGForgeFlow 

Could you review this fix please?

Thank you very much

File: purchase_open_qty/views/purchase_view.xml [hunk 1]
```diff
@@ -33,10 +33,10 @@
             <filter name="hide_cancelled" position="after">
                 <filter name="pending_qty_to_invoice"
                         string="Pending Qty to Bill"
-                        domain="[('pending_qty_to_invoice','=',True)]"/>
+                        domain="[('qty_to_invoice','>',0)]"/>
```
Comment (MiquelRForgeFlow) [inline purchase_open_qty/views/purchase_view.xml]:
```suggestion
                        domain="[('qty_to_invoice','>',0.0)]"/>
```
the same for the other

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM after taking into account Miquel's comment

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM :+1: Same comment as @MiquelRForgeFlow

**Comment by eantones** [comment]
Thanks! :-)

**Pull request merged**